client/src/components/Shipping/ShippingTable.jsx

```diff
@@ -0,0 +1,179 @@
+import React, { Fragment, useState } from 'react';
+import {
+  Checkbox,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Heading,
+  Button,
+  IconButton,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+} from '@chakra-ui/react';
+import { StyledTableRow, StyledTableHeader, StyledTableCell } from 'components/common/Table.jsx';
+import { TablePagination } from '@material-ui/core';
+import { shippingStates } from 'constants.js';
+import { ChevronDownIcon, DeleteIcon } from '@chakra-ui/icons';
+import AddShipmentForm from './AddShipmentForm.jsx';
+import useShippingTable from 'hooks/useShippingTable.jsx';
+import { NoResultImage } from 'components/common/Image.jsx';
+import { formatDate } from 'utils/dateFunctions.js';
+import Loader from 'components/common/Loader.jsx';
+import Head from 'next/head';
+
+const ShippingHeader = () => (
+  <>
+    <Head>
+      <title>ERP - Orders</title>
+    </Head>
+    <Heading fontSize={{ base: '12px', sm: '26px' }} textAlign="center" mt={5}>
+      Shippings
+    </Heading>
+  </>
+);
+const ShippingTable = () => {
+  const {
+    isSelected,
+    handleSelectAllClick,
+    handleOnChange,
+    handleDelete,
+    handleShpmtStateChange,
+    handleHover,
+    isLoading,
+    isSuccess,
+    data,
+    selected,
+  } = useShippingTable();
+
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  if (isSuccess && data.data.length === 0) {
+    return (
+      <>
+        <ShippingHeader />
+        <Heading size="xl" textAlign="center" mt={5}>
+          No shippings found.
+        </Heading>
+        <AddShipmentForm showButton={true} />
+
+        <NoResultImage />
```

Comment:
I think we can remove this image for this page because the map takes a lot of space already.